src/SFA.DAS.EmployerAccounts.Web/Views/Shared/_Layout.cshtml

```diff
@@ -228,11 +228,20 @@
             </div>
         }
     }
-            <main role="main" id="content">
-                @RenderSection("breadcrumb", required: false)
-                @Html.Partial(@"_SuccessMessage", Model as OrchestratorResponse)
-                @RenderBody()
-            </main>
+    <main role="main" id="content">
+        @RenderSection("breadcrumb", required: false)
+        @{
+            var userId = Html.ViewContext.RequestContext.HttpContext.GetOwinContext().Authentication.User.Claims.

```

Comment:
Might be best to keep all this inside the htmlHelper extension as we will be less likely to miss it when removing this code.